src/main/java/com/corndel/pixmate/drawings/Diagonal.java

```diff
@@ -0,0 +1,13 @@
+package com.corndel.pixmate.drawings;
+
+public class Diagonal {
+    public static void main(String[] args) {
+        StringBuilder line = new StringBuilder("*");
+
+        for (int i = 0; i < 5; i++) {
```

Comment:
I like this approach and the `InvertedDiagonal`, good problem-solving!

 How do you think you could make this more dynamic to create more then 5?